### ISSUE TITLE
Space comparison

### ISSUE THREAD

**Issue by pzjp** [pi-base/data#1157]
I got a loosely thought on new ways of exploring the base.
The most precise tool I have in mind is **space comparison** - ability to see all traits that distinguish given spaces. Especially seeing "similar spaces" (e.g. [S83](https://topology.pi-base.org/spaces/S83)/[S84](https://topology.pi-base.org/spaces/S84); [S60](https://topology.pi-base.org/spaces/S60)/[S61](https://topology.pi-base.org/spaces/S61)) an interest in what topological properties tell them apart may appear. One may say _it's enough to open two browser tabs side by side_ but the convenience of such approach is limited, isn't it?

A further extension could be a view of more than two spaces at once, but probably it's not that much usefull, yet creating many questions on UI details.

As a more abstract concept I considered also possibility of measuring a "distance" between two spaces (a number of traits that are different) and presenting the most "similar" spaces to the given one (I guess such search results may be surprising). But I imagine this being seen as a quite random query to the base (I can think of something weirder, though). Maybe someone has a better idea of this kind?

Of course this kind of improvements are of not very high priority but I guess it is worth consideration. Especially they do not require rebuilding the data itself like the meta-properties do.

**Comment by prabau** [comment]
I agree long term it would be nice a comparison feature between two spaces.

@StevenClontz
